applications/openshift/api-server/api_server_admission_control_plugin_AlwaysAdmit/rule.yml

```diff
@@ -1,28 +1,25 @@
 documentation_complete: true
 
-title: 'Ensure that the admission control plugin AlwaysAdmit is not set'
+title: 'Ensure that the admission control plugin AlwaysAdmit is not Configured'
```

Comment:
```suggestion
title: 'Ensure that the admission control plugin AlwaysAdmit is not configured'
```

Reply:
This PR is a WIP, but I am actually thinking that some of the titling will change to match what we do with the rest of the content.